h2o-r/h2o-package/R/stackedensemble.R

```diff
@@ -44,6 +46,7 @@ h2o.stackedEnsemble <- function(x, y, training_frame,
                                 metalearner_fold_assignment = c("AUTO", "Random", "Modulo", "Stratified"),
                                 metalearner_fold_column = NULL,
                                 keep_levelone_frame = FALSE,
+                                seed = -1,
```

Comment:
This is not really directly related to this PR, but I am noticing this now... Can we slightly change the order of the params?  Not sure how much work that is in `gen_R.py` but it would be great to better group related arguments together.  

Current:
```
h2o.stackedEnsemble <- function(x, y, training_frame,
                                model_id = NULL,
                                validation_frame = NULL,
                                base_models = list(),
                                metalearner_algorithm = c("AUTO", "glm", "gbm", "drf", "deeplearning"),
                                metalearner_nfolds = 0,
                                metalearner_fold_assignment = c("AUTO", "Random", "Modulo", "Stratified"),
                                metalearner_fold_column = NULL,
                                keep_levelone_frame = FALSE,
                                seed = -1,
                                metalearner_params = NULL) 
```
proposed (let's move `metalearner_params` up):
```
h2o.stackedEnsemble <- function(x, y, training_frame,
                                model_id = NULL,
                                validation_frame = NULL,
                                base_models = list(),
                                metalearner_algorithm = c("AUTO", "glm", "gbm", "drf", "deeplearning"),
                                metalearner_params = NULL ,
                                metalearner_nfolds = 0,
                                metalearner_fold_assignment = c("AUTO", "Random", "Modulo", "Stratified"),
                                metalearner_fold_column = NULL,
                                keep_levelone_frame = FALSE,
                                seed = -1) 
```

Reply:
This is technically breaking API since named params are not needed.

Reply:
If its okay with you, I would like to do this in another PR